examples/reference/pandas/bar.ipynb

```diff
@@ -55,13 +53,6 @@
     "autompg.head()"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Note that the columns `yr`, `origin`, `name` and `mfr` are categorical.Note that the columns `yr`, `origin`, `name` and `mfr` are categorical."
```

Comment:
Duplicate sentences, and almost all the data in this dataset is actually categorical, I think